File: backend/pawnscan/src/main/java/edu/cit/garbo/pawnscan/features/businessprofile/dto/BusinessProfileResponse.java
package edu.cit.garbo.pawnscan.features.businessprofile.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.time.LocalDateTime;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class BusinessProfileResponse {

    private Long userId;
    private String businessName;
    private String businessAddress;
    private String permitNumber;
    private boolean isVerified;
    private boolean isRejected;
    private String rejectionReason;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
}

File: mobile/app/src/main/java/com/cit/pawnscan/features/auth/api/BusinessProfileResponse.kt
package com.cit.pawnscan.features.auth.api

data class BusinessProfileResponse(
    val businessName: String?,
    val businessAddress: String?,
    val permitNumber: String?,
    val verified: Boolean?,
    val isVerified: Boolean?,
    val rejected: Boolean?,
    val isRejected: Boolean?,
    val rejectionReason: String?
)
